### ISSUE TITLE
[FEATURE] : Add "Emojii" Feature On Capture Section in Separate New Post Card 

### ISSUE THREAD

**Issue by NarendraKoya999** [narayan954/dummygram#463]
### Issue Description:
-  **Add Emojii Feature On Capture Section in Separate New Post Card ( Respective Feature Image Shared in Screenshots Section  )**

> How It Works ??
- **After Signing with Particular E-Mail. Click on New Post Option on Top of The Header Section.**
- **Then, Separate New Post Design Card Shown there.**
- **Observe Caption Item and When Click on that Text Area, Only Possible to write Some Text.**
- **With Extra Meaningful Information, I suggest to Add Emojii Feature on Capture Text Area.** 

### Screenshot of Above Requested Feature and Where to add this ??

<img width="388" alt="dummygram-2" src="https://github.com/narayan954/dummygram/assets/84512164/c0116810-7cdf-4b95-a32e-ba219bd810d6">

### NOTE:
- **Generally, In Latest Trends, Emojii Feature is Thoroughly Implemented in Some Social Websites and Others.**
- **But, Which Area is to Best Fit for adding this Emojii Feature is also Most Important.**
- **That's Why I recommend This Feature.**

**Comment by narayan954** [comment]
Yes sure, also please adjust to the new UI, as things have changed quite a bit!

**Comment by NarendraKoya999** [comment]
Hi @narayan954 , Please Assign this Issue To Others. Who Can Develop to this Feature !!

**Comment by narayan954** [comment]
> Hi @narayan954 , Please Assign this Issue To Others. Who Can Develop to this Feature !!

yeah sure!

**Comment by NarendraKoya999** [comment]
Hi @narayan954 . If Possible, I will take up the Issue. Because of It's Still Open ??

**Comment by NarendraKoya999** [comment]
Thank you @narayan954
